### PR TITLE
Fix e2e flakiness for setup test

### LINKED ISSUE #9193: Fix flaky Playwright Tests
## Motivation

- Playwright runs have become increasingly flaky lately
- We're increasingly seeing Playwright failures and deciding to re-run the tests rather than investigate
    - This slows development
    - Means real failures are not investigated immediately
    
## Acceptance Criteria

- [ ] Recent flaky tests are identified
- [ ] Solutions to prevent tests from failing are implemented
- [ ] If there are any best practices we should be using to prevent flaky tests, this is shared with the team

### PR DESCRIPTION
**Pull request by fungairino**
## What does this PR do?

- Closes #9193

## Future Work

- Do we want to track how often this particular type of linking issue happens? If so, what would be a good way to do so?

## Checklist

- [ ] This PR requires a security review
- [ ] This PR introduces a new library: double check it's MIT/Apache2/permissively licensed
- [ ] This PR requires a node/npm version update: let the team know on #engineering
- [ ] This PR requires a documentation change (link to old docs)
- [ ] This PR requires a tutorial update (link to old tutorials)
- [ ] This PR requires a feature flag
- [ ] This PR requires a environment variable change
- [x] Added jest or playwright tests and/or storybook stories

For more information on our expectations for the PR process, see the
[code review principles doc](https://www.notion.so/pixiebrix/Code-Review-Principles-1ce7276b82a84d2a995d55ad85e1310d?pvs=4)

### CODE REVIEW

File: end-to-end-tests/setup/utils.ts [hunk 1]
```diff
@@ -28,6 +28,19 @@ export const openExtensionConsoleFromAdmin = async (
...
   await expect(async () => {
+    const openExtensionConsoleButton = adminPage
+      .locator("button")
+      .filter({ hasText: "Open Extension Console" });
+
+    try {
+      await expect(openExtensionConsoleButton).toBeEnabled({ timeout: 5000 });
+    } catch (error) {
+      // Sometimes the app fails to link with the extension and the button remains disabled.
+      // In this case, reload the page to force the app to retry linking.
```
Comment (mnholtz) [inline end-to-end-tests/setup/utils.ts:39]: Out of curiosity, does the "success" toast notification appear in these cases? I'm wondering if it's an issue with sending the message to link the extension or something after that

File: playwright.config.ts [hunk 2]
```diff
@@ -98,7 +98,7 @@ export default defineConfig<{ chromiumChannel: string }>({
     ["json", { outputFile: "./end-to-end-tests/.report/report.json" }],
   ],
   // /* Repeat each test N times. Useful for catching flaky test. */
-  // repeatEach: 3,
+  // repeatEach: 6,
```
Comment (mnholtz) [inline playwright.config.ts:101]: Do we want to keep the recommendation at 6?

**Review by mnholtz — APPROVED** [review]

File: playwright.config.ts (hunk 2 above)
Comment (fungairino) [reply playwright.config.ts:101]: it's not a really a recommendation, just a placeholder value that you can adjust.

File: end-to-end-tests/setup/utils.ts (hunk 1 above)
Comment (fungairino) [reply end-to-end-tests/setup/utils.ts:39]: Yea it does show the toast notification.
Screenshot 2024-10-11 at 11 28 58 AM

See this run for the example:
` ./scripts/show-pr-e2e-report.sh -r 11281686835`

**Pull request merged**